Added icons to buttons.

ClemiNeko opened the pull request:
Changed texts style to bold/italic.
Changed app name from "HumanAnime" to "Transformer".
Added a default image to the MainActivity ImageView.

keyofdeath commented:
No probleme

keyofdeath commented:
![2hen5i](https://user-images.githubusercontent.com/16309783/46403128-f1c64e00-c701-11e8-8ef7-23d5d468b8e9.jpg)

Commit:
Added icons to buttons.
Changed texts style to bold/italic.
Changed app name from "HumanAnime" to "Transformer".
Added a default image to the MainActivity ImageView.
